.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
+    "trailingComma": "none"
```

Comment:
I like trailing commas :(

Reply:
p.s. trailing commas are [a security matter](https://certitude.consulting/blog/en/invisible-backdoor/)

Reply:
Haha me too i was just adding this to test it out locally, ill leave it as an empty .prettierrc for now